ecosystem/op-deployer.md

```diff
@@ -101,7 +101,7 @@ baseFeeVaultRecipient = "0xbc4a9110dad00b4d9fb61743598848ddda6eeb03"
 l1FeeVaultRecipient = "0x6b0c2542fa2cadced5c7f64ef6fb9ebbce7630ff"
 sequencerFeeVaultRecipient = "0xb4e5b724bbc54c95d292613d956871281120ead6"
 
-# Address of OPSM so that other addresses can be retrieved from on-chain data
+# Address of OPCM so that other addresses can be retrieved from on-chain data
 opStackManagerAddress = "0x79c6c6b1844e3db7c30107f189cfb095bd2c4b5d"
```

Comment:
Approved too quick, caught this one after

Reply:
https://github.com/ethereum-optimism/design-docs/pull/90/commits/f69264f39e80d088ea8767ba9cbd6e16793365df